cvescan/__main__.py

```diff
@@ -308,11 +325,11 @@ def main():
         LOGGER.debug("Manifest package count is %s" % package_count)
 
     (cve_list_all_filtered, cve_list_fixable_filtered) = \
-        scan_for_cves(now, opt.verbose_oscap_options, opt.oval_file,
-                opt.scriptdir, opt.xslt_file, opt.extra_sed, opt.priority)
+        scan_for_cves(sysinfo.process_start_time, opt.verbose_oscap_options, opt.oval_file,
+                sysinfo.scriptdir, sysinfo.xslt_file, opt.extra_sed, opt.priority)
 
-    if snap_user_common == None or len(snap_user_common) == 0:
-      LOGGER.debug("Full HTML report available in %s/%s" % (opt.scriptdir, REPORT))
+    if not sysinfo.is_snap:
```

Comment:
this condition can probably be removed to always LOGGER.debug the full location of the html report

Reply:
There are a few conditions like this that will all go away when I debug log the entire set of relevant options at initialization.